packages/PlanLimitsUI/src/pages/Limits/WaterQuantity.spec.tsx

```diff
@@ -0,0 +1,12 @@
+import { expect } from 'vitest';
+import waterQuantity from './WaterQuantity';
+
+describe('.waterQuantity', () => {
+  test('should format number with units', () => {
```

Comment:
Only minor, but I notice we're using a mix of describe/test and describe/it syntax - should we pick one to standardise on? I have a slight preference for `it`, but don't feel too strongly either way.

Reply:
`it` it is ... I've changed them all to be `it`